src/lightning_app/testing/testing.py

```diff
@@ -249,7 +249,7 @@ def run_app_in_cloud(app_folder: str, app_name: str = "app.py") -> Generator:
         admin_page.goto(f"{Config.url}/{Config.username}/apps")
         try:
             # Closing the Create Project modal
-            button = admin_page.locator('button:has-text("Cancel")')
```

Comment:
To make the e2e pass, maybe we can allow both either `Cancel` or `Continue` ?

`'button:has-text("Cancel"), button:has-text("Continue")'`

Reply:
@manskx Ah yes, I was actually going to try and do this but I couldn't figure out the sytax for Python+Playwright. Let me update the code to do that instead.

Reply:
It will break if we have both buttons, hopefully not 😅